fe/fe-core/src/main/java/com/starrocks/clone/TabletSchedCtx.java

```diff
@@ -755,6 +755,10 @@ public void resetDecommissionedReplicaState() {
                 && decommissionedReplica.getState() == ReplicaState.DECOMMISSION
                 && decommissionedReplicaPreviousState != null) {
             decommissionedReplica.setState(decommissionedReplicaPreviousState);
+            decommissionedReplica.setWatermarkTxnId(-1);
```

Comment:
1. Why does it can prevent the delete replica?
2. If the balance comes from A, B -> A, A', B. The A and A' succeed, but B not. If not delete A, will it be violate the quorum?

Reply:
1. because if the txn water mark is not reset(still the old one), `deleteReplicaInternal` won't wait for the newest txn water mark.
2. no, A' is not counted, quorum won't match.